src/modules/imageresizer/ui/Views/InputPage.xaml

```diff
@@ -216,99 +195,102 @@
                         </DataTemplate>
                     </ComboBox.ItemTemplate>
                 </ComboBox>
-
             </Grid>
 
+            <!--  CheckBoxes  -->
             <StackPanel
                 Grid.Row="1"
-                VerticalAlignment="Top"
-                Orientation="Vertical">
+                Margin="16,12,16,0"
```

Comment:
![image](https://github.com/microsoft/PowerToys/assets/9866362/f1642068-e792-4461-ae2b-6c9da4b74c0c)

There is a difference in spacing between the top part and bottom part

Reply:
Can't look at the boxes here - remember TextWrapping is added.